pfcpiface/bess.go

```diff
@@ -278,6 +278,12 @@ func (b *bess) setUpfInfo(u *upf, conf *Conf) {
 		u.simInfo = simInfo
 	}
 
+	u.ippool_cidr = conf.CPIface.UeIPPool
```

Comment:
change all naming to not have underscore. I thought one of the linting tool checks and throws warning

Reply:
done